classes/class-wc-connect-nux.php

```diff
@@ -364,17 +374,27 @@ public function get_feature_list_for_country( $country ) {
 			$supports_rates  = in_array( $country, array( 'US', 'CA' ) );
 			$supports_labels = ( 'US' === $country );
 
-			if ( $supports_stripe && $supports_taxes && $supports_rates && $supports_labels ) {
+			$is_stripe_active = is_plugin_active( 'woocommerce-gateway-stripe/woocommerce-gateway-stripe.php' );
+			$stripe_settings  = get_option( 'woocommerce_stripe_settings', array() );
+			$is_stripe_ready  = $is_stripe_active && isset( $stripe_settings['enabled'] ) && 'yes' === $stripe_settings['enabled'];
+
+			$is_ppec_active = is_plugin_active( 'woocommerce-gateway-paypal-express-checkout/woocommerce-gateway-paypal-express-checkout.php' );
+			$ppec_settings  = get_option( 'woocommerce_ppec_paypal_settings', array() );
+			$is_ppec_ready  = $is_ppec_active && isset( $ppec_settings['enabled'] ) && 'yes' === $ppec_settings['enabled'];
```

Comment:
@jeffstieler I noticed from your test today that the WCS banner is missing a mention of payments in the case that `'enabled'` isn't set, even though WCS itself will make sure it is initialized to `'yes'`. Will modify this to treat the unset case as if it were enabled.